Gemfile

```diff
@@ -70,7 +70,6 @@ group :test, :development do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
   gem 'bullet'
   gem 'capybara'
-  gem 'devise'
   gem 'ffi'
   gem 'rails-controller-testing'
   gem 'rspec-rails', '~> 5.0.2'
```

Comment:
- [ ] It would be better if our PR describes the project with a title and a short summary with bullet points. In this way, our repo will look professional